root.json

```
@@ -2,6 +2,7 @@
     "2FAuth": "2FAuth.json",
     "AdGuard Home": "adguard.json",
     "Airsonic Advanced": "airsonic-advanced.json",
+    "audiobookshelf": "audiobookshelf.json",
```

```suggestion
    "Audiobookshelf": "audiobookshelf.json",
```

As per the new Capitalisation in our Rock-on naming.

---

OK thanks.